satpy/tests/reader_tests/test_eps_l1b.py

```diff
@@ -160,6 +163,63 @@ def mock_getitem(key):
             sun_zen_np2 = np.array(avhrr_reader.sun_zen)
             assert np.allclose(sun_zen_np1, sun_zen_np2)
 
+
+class TestWrongEPSL1B(TestCase):
+    """Test the filehandler on a corrupt file."""
+
+    @pytest.fixture(autouse=True)
+    def inject_fixtures(self, caplog):
+        """Inject caplog."""
+        self._caplog = caplog
+
+    def setUp(self):
+        """Set up the tests."""
+        # ipr is not present in the xml format ?
+        structure = [(1, ('mphr', 0)), (1, ('sphr', 0)), (11, ('ipr', 0)),
+                     (1, ('geadr', 1)), (1, ('geadr', 2)), (1, ('geadr', 3)),
+                     (1, ('geadr', 4)), (1, ('geadr', 5)), (1, ('geadr', 6)),
+                     (1, ('geadr', 7)), (1, ('giadr', 1)), (1, ('giadr', 2)),
+                     (1, ('veadr', 1)), (1080, ('mdr', 2))]
```

Comment:
How about adding `self.scanlines = 1080` and use that?

Reply:
Good idea

Reply:
done